go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	google.golang.org/grpc v1.59.0
 	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.3.0
 	google.golang.org/protobuf v1.31.0
+	istio.io/tools v0.0.0-20231026140029-4cb5f8a9b39a
```

Comment:
I have no idea why `make check-diff` is failing in CI. Everything seems fine locally. I wonder if it's something architecture related?